config_read.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push('../fluentd/lib')
```

Comment:
I've copied over the entire repository (not just specific files) into my project folder for now, but will research alternatives too.

Reply:
Have we decided how to package this tool? Using bundler with Gemfile/gemspec is one option. It can also just do `gem install fluentd`, depending on how we package it.

The simplest way is to use a Ruby container with gem Fluentd pre-installed and use Docker to run the tool. But that may not meet all the requirements we are targeting.

Reply:
Some example: https://medium.com/@gratefulcheddar/rubygems-101-how-to-use-a-third-party-gem-in-your-ruby-application-37249529dbc7

Reply:
Not yet, but I think we can start with gemfile/gemspec. As Mihika is doing experimentation on whether to include fluentd config parser code, we can review the packaging part later.